packages/Checkboxes/src/Checkboxes.tsx

```diff
@@ -99,7 +101,11 @@ const CheckBoxes: FunctionComponent<ICheckBoxesProps> = ({
               disabled={disabled}
               id={id}
               data-testid={id}
-              label={label}
+              label={
+                <>
+                  {label} {tooltip && <span className="tooltip">{tooltip}</span>}
```

Comment:
Doesn't look quite right at the moment, the tooltip should just be shown on the label not every option. we don't need this one yet because we have it on the [parent](https://github.com/thebyte9/blaze-components-react/blob/4e20cb30fdd175be3870145564c9a4dc803821b5/packages/Checkboxes/src/Checkbox.tsx)

Also it is not using the Tooltip component.